components/Form.js

```diff
@@ -1,15 +1,18 @@
 import { useRouter } from "next/router";
 import { styled } from "styled-components";
+import { StyledButton } from "./StyledButton";
 
-export default function Form({ submitNewCard }) {
+export default function Form({ submitNewCard, card, updateCard }) {
   const router = useRouter();
 
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
 
-    submitNewCard(data.name, data.icon);
+    card
+      ? updateCard(data.name, data.icon, card.id)
+      : submitNewCard(data.name, data.icon);
 
     event.target.reset();
     router.push("/");
```

Comment:
Oopsie, I meant this line.

Reply:
Hey Miejke, thanks for your comment, Yes I changed my mind - so that you can "Play" ;) I am not sure if / or /edit is better ... I will think about that, may be /edit makes more sense... not sure

Reply:
Okay sure, I like /homepage but maybe you get feedback from other reviewers about that :)